app/views/dashboard/index.njk

```diff
@@ -5,12 +5,18 @@ Dashboard - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK
 {% endblock %}
 
 {% block mainContent %}
-  <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-l govuk-!-margin-top-6 first-steps__title">Dashboard</h1>
-  </div>
-
-  {% if paymentMethod === 'card' %}
-    {% include "./_activity.njk" %}
+  {% if account.paymentMethod === 'direct debit' and account.type === 'live' and account.isConnected === false %}
```

Comment:
I think we can leave out the check that the account is 'live', as we're able to connect sandbox accounts. We do need a check that the payment provider is GoCardless though.